src/main/java/com/google/devtools/build/lib/remote/RemoteSpawnRunner.java

```diff
@@ -304,6 +304,10 @@ private SpawnResult downloadAndFinalizeSpawnResult(
       SpawnExecutionContext context,
       RemoteOutputsMode remoteOutputsMode)
       throws ExecException, IOException, InterruptedException {
+    // Ensure that when using dynamic spawn strategy that we are the only ones writing to the
```

Comment:
Reword: "Ensure that we are the only ones writing to the output files when using the dynamic spawn strategy."

Also not sure why the bug link in the code is necessary. Yes, there was a bug that's fixed with this change. Reference it in the commit description instead?